generators/ruby/faraday/templates/configuration.mustache

```diff
@@ -0,0 +1,508 @@
+    #
+    # @return [String]
+    attr_accessor :password
+
+    # Defines the access token (Bearer) used with OAuth2.
+    attr_accessor :access_token
+
+    # Defines a Proc used to fetch or refresh access tokens (Bearer) used with OAuth2.
+    # Overrides the access_token if set
+    # @return [Proc]
+    attr_accessor :access_token_getter
+
+    # Set this to return data as binary instead of downloading a temp file. When enabled (set to true)
+    # HTTP responses with return type `File` will be returned as a stream of binary data.
+    # Default to false.
+    attr_accessor :return_binary_data
+
+    # Set this to enable/disable debugging. When enabled (set to true), HTTP request/response
+    # details will be logged with `logger.debug` (see the `logger` attribute).
+    # Default to false.
+    #
+    # @return [true, false]
+    attr_accessor :debugging
+
+    # Defines the logger used for debugging.
+    # Default to `Rails.logger` (when in Rails) or logging to STDOUT.
+    #
+    # @return [#debug]
+    attr_accessor :logger
+
+    # Defines the temporary folder to store downloaded files
+    # (for API endpoints that have file response).
+    # Default to use `Tempfile`.
+    #
+    # @return [String]
+    attr_accessor :temp_folder_path
+
+    # The time limit for HTTP request in seconds.
+    # Default to 0 (never times out).
+    attr_accessor :timeout
+
+    # Set this to false to skip client side validation in the operation.
+    # Default to true.
+    # @return [true, false]
+    attr_accessor :client_side_validation
+
+{{#isTyphoeus}}
+{{> configuration_typhoeus_partial}}
+{{/isTyphoeus}}
+{{#isFaraday}}
+{{> configuration_faraday_partial}}
+{{/isFaraday}}
+{{#isHttpx}}
+{{> configuration_httpx_partial}}
+{{/isHttpx}}
+
+    attr_accessor :inject_format
+
+    attr_accessor :force_ending_format
+
+    def initialize
+      @scheme = '{{scheme}}'
+      @host = '{{host}}{{#port}}:{{{.}}}{{/port}}'
+      @base_path = '{{contextPath}}'
+      @server_index = nil
+      @server_operation_index = {}
+      @server_variables = {}
+      @server_operation_variables = {}
+      @api_key = {}
+      @api_key_prefix = {}
+      @client_side_validation = true
+      {{#isFaraday}}
+      @ssl_verify = true
+      @ssl_verify_mode = nil
+      @ssl_ca_file = nil
+      @ssl_client_cert = nil
+      @ssl_client_key = nil
+      @middlewares = Hash.new { |h, k| h[k] = [] }
+      @configure_connection_blocks = []
+      {{! Modified default timeout to 30}}
+      @timeout = 30
+      # return data as binary instead of file
+      @return_binary_data = false
+      @params_encoder = nil
+      {{/isFaraday}}
+      {{#isTyphoeus}}
+      @verify_ssl = true
+      @verify_ssl_host = true
+      @cert_file = nil
+      @key_file = nil
+      @timeout = 0
+      @params_encoding = nil
+      {{/isTyphoeus}}
+      {{#isHttpx}}
+      @ssl = nil
+      @proxy = nil
+      @timeout = 60
```

Comment:
This should be removed because we are setting to 30 above

Reply:
Shouldn't it take only the isFaraday case?

Reply:
Ah sorry, I missed the mustache conditional